WPFHospitalEditor/Controller/DoctorServerController.cs

```diff
@@ -20,9 +20,19 @@ public Doctor GetDoctorById(int doctorId)
             return doctorServerService.GetDoctorById(doctorId);
         }
 
+        public IEnumerable<DoctorDto> GetFilteredDoctors(string name)
+        {
+            return doctorServerService.GetFilteredDoctors(name);
+        }
+
         public IEnumerable<DoctorDto> GetAllSpecialists()
         {
             return doctorServerService.GetAllSpecialists();
         }
+
+        public IEnumerable<DoctorDto> GetFilteredSpecialists(string name)
+        {
+            return doctorServerService.GetFilteredSpecialists(name);
+        }
```

Comment:
Same problem as above

Reply:
You also have "filter" in names for medication and patients.

Reply:
I renamed all of them.